### PR TITLE
Email optional branch

### PR DESCRIPTION
**Pull request by danward79**
Updated to make email link optional

### CODE REVIEW

**Comment by danward79** [comment]
Hi @humboldtux, 
I'll submit the other pull requests in the same manner, if this one is acceptable. Just to make sure I have the process correct.

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -5,14 +5,16 @@
...
                   <ul class="list-inline text-center">
-                    <li>
-                      <a href="mailto:{{ .Site.Params.email }}">
-                        <span class="fa-stack fa-lg">
-                          <i class="fa fa-circle fa-stack-2x"></i>
-                          <i class="fa fa-envelope-o fa-stack-1x fa-inverse"></i>
-                        </span>
-                      </a>
-                    </li>
+                    {{ with .Site.Params.Email }}
```
Comment (humboldtux) [inline layouts/partials/footer.html]: Argh, this is not working anymore, the param is `email`, all lowercases, you should have used:
` {{ with .Site.Params.email }}` or `{{ if isset .Site.Params "email" }}` like in your first PR.

We are almost there @danward79

**Comment by danward79** [comment]
No worries @humboldtux, fixed now.

**Comment by humboldtux** [comment]
Thank you very much @danward79 
You can create other PRs in their own branch this way if you are willing to.

**Pull request merged**